src/CattyTests/PlayerEngine/CBSpriteNodeTests.swift

```diff
@@ -53,15 +53,15 @@ final class CBSpriteNodeTests: XCTestCase {
     }
 
     func testPositionX() {
-        spriteNode.catrobatPositionX = 10
+        spriteNode.catrobatPosition = CBPosition(x: 10, y: spriteNode.catrobatPosition.y)
         XCTAssertEqual(PositionXSensor.convertToRaw(userInput: 10, for: spriteNode.spriteObject),
                        Double(spriteNode.position.x),
                        accuracy: epsilon,
                        "SpriteNode catrobatPosition not correct")
     }
 
     func testPositionY() {
-        spriteNode.catrobatPositionY = 20
+        spriteNode.catrobatPosition = CBPosition(x: spriteNode.catrobatPosition.y, y: 20)
```

Comment:
Please correct the x position and squash your commits.

Reply:
Done!